### ISSUE TITLE
Unable to serialize a resnet model to a saved model

### ISSUE THREAD

**Issue by cbodenst** [broadinstitute/keras-resnet#64]
The newest keras version allow us to save models not only as .h5 keras files, but also as saved model.

However, custom layers must be correctly declared to support this.

I.e the BatchNormalization layer should define its signature for "call" properly as:
`def call(self, inputs, **kwargs):`
instead of

`def call(self, *args, **kwargs):`

You can read more about this issue here:
https://github.com/tensorflow/tensorflow/issues/38384
